tests/roles/ceph_backend_configuration/tasks/main.yaml

```diff
@@ -0,0 +1,54 @@
+- name: set shell vars for stopping openstack services
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.set_fact:
+    ceph_backend_configuration_shell_vars: |
+      CEPH_SSH="{{ controller1_ssh }}"
+      CEPH_KEY=$($CEPH_SSH "cat /etc/ceph/ceph.client.openstack.keyring | base64 -w 0")
+      CEPH_CONF=$($CEPH_SSH "cat /etc/ceph/ceph.conf | base64 -w 0")
+
+- name: create ceph-conf-files secret
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    {{ ceph_backend_configuration_shell_vars }}
+
+    oc apply -f - <<EOF
```

Comment:
I thought we could simplify this part by running ` oc create secret generic ceph-conf-files --from-file=/etc/ceph/ceph.client.openstack.keyring --from-file=/etc/ceph/ceph.conf` but I see you're grabbing the content by sshing to controller1 (where I assume Ceph is deployed).
The alternative would be:
- slurp files from controller1
- create secrets from files

This way you can catch issues in the file content and you don't have to convert them in `base64`, but in general I'm ok to keep this approach.

Reply:
That makes sense, it's probably better to use --from-file, for now i created an issue to track it since i didn't have time to tweak & test this variant: https://github.com/openstack-k8s-operators/data-plane-adoption/issues/125